src/StatsBase.jl

```diff
@@ -2,8 +2,8 @@ __precompile__(true)
 
 module StatsBase
     using Compat
-    import Compat.String
-    using ArrayViews
+    import Compat: String, view
```

Comment:
Looks like the explicit `import` actually is necessary here; it fails on 0.4 because it can't find `view`.